tests/svgWriter.test.js

```diff
@@ -227,182 +228,16 @@ describe('svgWriter', function (){
-            compareResults('svgFx-inner-glow');
-        });
-
-        it("should show radial gradients", function () {
-            compareResults('svgGradient-radial');
-        });
-
-        it("should show scaled gradients", function () {
-            compareResults('gradient-scale');
-        });
-
-        it("should show reflected gradients", function () {
-            compareResults('svgGradient-reflected');
-        });
-
-        it("should show scaled, reflected gradients", function () {
-            compareResults('gradient-scale-reflected');
-        });
-
-        it("should show reversed gradients", function () {
-            compareResults('gradient-reverse');
-        });
-
-        it("should show stroke style", function () {
-            compareResults('stroke-style');
-        });
-
-        it("should show fx effects on grouping layers", function () {
-            compareResults('group');
-        });
-
-        it("should support different radii for radial gradients in layer space", function () {
-            compareResults('radial-gradient-angle-layer');
-        });
-
-        it("should support different radii for radial gradients in global space", function () {
-            compareResults('radial-gradient-angle-global');
-        });
-
-        it("should support different radii for linear gradients in layer space", function () {
-            compareResults('linear-gradient-angle-layer');
-        });
-
-        it("should support different radii for linear gradients in layer space part 2", function () {
-            compareResults('linear-gradient-angle-layer-2');
-        });
-
-        it("should support different radii for linear gradients in global space", function () {
-            compareResults('linear-gradient-angle-global');
-        });
-
-        it("should support different radii for linear gradients in layer space part 2", function () {
-            compareResults('linear-gradient-angle-global-2');
-        });
-
-        it("should support text styling", function () {
-            compareResults('text-styling');
-        });
-
-        it("should support text on path", function () {
-            compareResults('text-on-path');
-        });
-
-        it("should support text transformation", function () {
-            compareResults('text-transform');
-        });
-
-        it("should differ between local and gloval lighting", function () {
-            compareResults('light-global-local');
-        });
-
-        it("should support fx stroke", function () {
-            compareResults('stroke-fx');
-        });
-
-        it("should support gradient overlay in combination with color overlay", function () {
-            compareResults('gradient-color-overlay');
-            
-            _isLastTest = true;
-        });
 
+        for (var i = 0, end = database.length; i < end; i++) {
+            var test = database[i].test;
+                desc = database[i].desc;
+            it(desc, function () {
```

Comment:
the variable 'test' is not stable and will change before the test closure executes.  using this loop each test will execute the comparison of the last test in the list.

define a function that takes test and desc variables and call "it(..." from within that function closure to make the test and desc references stable